packages/@okta/vuepress-site/docs/guides/authenticators-web-authn/main/index.md

```diff
@@ -31,6 +31,8 @@ To be resistant to phishing attacks, a local communication channel should exist
 
 Besides being resistant to phishing attacks, WebAuthn can drastically reduce the sign-in friction by allowing passwordless sign-ins during reauthentication use cases. For example, mobile banking apps use this type of sign-in today allowing iPhone users to sign-in using only Face ID after the first sign-in with a password.
 
+> **Note**: For detailed information on the Webauthn standard, including an up-to-date list of supported browsers, see [webauthn.me](https://a0.to/webauthnme-okta-docs).
```

Comment:
```suggestion
> **Note**: For detailed information on the WebAuthn standard, including an up-to-date list of supported browsers, see [webauthn.me](https://a0.to/webauthnme-okta-docs).
```

The official spelling is `WebAuthn`

Reply:
This isn't a file that I changed... I'll have to check the commit to make sure it catches only my files.